all/entrypoint.sh

```diff
@@ -143,6 +143,10 @@ case $GEN_LANG in
     "java")
         GEN_STRING="--grpc_out=$OUT_DIR --${GEN_LANG}_out=$OUT_DIR --plugin=protoc-gen-grpc=`which protoc-gen-grpc-java`"
         ;;
+    "node")
+        SERVICE_NAME=$(basename $PROTO_DIR)
+        GEN_STRING="--grpc_out=$OUT_DIR --js_out=import_style=commonjs,binary:$OUT_DIR --plugin=protoc-gen-grpc=`which grpc_${PLUGIN_LANG}_plugin`"
```

Comment:
Question for you - how useful are these extra items (i.e. `import_style=commonjs,binary`)? Do you think that other people in the community might want different ones?

Reply:
I can only get commonjs style to output consistent files when using gRPC. The binary flag seems "standard". It's possible that people may want other combinations, but these ones seem pretty consistent with what happens for other languages in the image.